tests/crud.test.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @jest-environment node
+ */
+
+const request = require('supertest');
+const app = require('../server/index.js');
+const db = require('../database/index.js');
```

Comment:
May be left over from previous tests but this doesn't seem to be used at the moment in these tests.

Reply:
Good call! I removed the unused files.